### ISSUE TITLE
Pull requests tab only shows pull requests from upstream repo

### ISSUE THREAD

**Issue by DenverCoder1** [desktop/desktop#14127]
### Describe the bug

If you fork a repository and clone it in GitHub Desktop, the Pull Requests tab does not show pull requests in the forked repository, but rather only pull requests appearing in the repo that was forked

### Version & OS

Version 2.9.9-linux2 (x64), Arch Linux

### Steps to reproduce the behavior

1. Open a repository which is a fork of another repository and has pull requests
2. Click on the branch pulldown menu
3. Switch to the Pull requests tab
4. See that Pull Requests from the fork do not appear

### Expected behavior

Pull requests from upstream and pull requests from the fork should be listed

### Actual behavior

Only upstream prs appear

### Screenshots

In this example, the cloned repo is DenverCoderOne/nextcord. The pull requests tab only shows pull requests in nextcord/nextcord and not DenverCoderOne/nextcord.

![image](https://user-images.githubusercontent.com/20955511/157362960-a2eb776c-5ff5-4685-ac67-2f2983b42626.png)

![image](https://user-images.githubusercontent.com/20955511/157362996-567801bc-a324-400b-998f-c8700097d0a7.png)

**Comment by niik** [comment]
Hi @DenverCoder1, when you first set up your fork you should have gotten a prompt asking you how you would like to use your fork. If it was for your own purposes or for contributing upstream. That setting governs what PRs are shown in the list.

Fortunately it's easy enough to change by opening repository settings and selecting the "Fork Behavior" tab

<img width="620" alt="image" src="https://user-images.githubusercontent.com/634063/157523065-f11f6e1d-ee8a-49ed-bc75-a5cd3cacaa79.png">

Hope that helps! I'll close this issue as it appears to be working as intended but if I've missed something just let me know and we can reopen

**Comment by DenverCoder1** [comment]
@niik Thanks for the response! That does help.

As a feature suggestion, is it possible to make it show pull requests for both?

**Comment by niik** [comment]
@DenverCoder1 Thanks for reporting back, glad to hear that was helpful.

Anything is possible but given that this feature and behavior has existed for quite some time now and this being the first request (that I know of) to change it to display both I think we'd be very reluctant to make such a change.

I would love to know more about your workflow though, in what scenario do you wish to both contribute upstream as well as use PRs inside of the fork?

Our hypothesis was that the vast majority of users would either fork a repository in order to contribute back to it or in order to more literally "fork" the repository and continue development on their own. A mix between the two was thought to be rare.

**Comment by DenverCoder1** [comment]
@niik I can understand that it would likely be a rare use case. Typically, when contributing to a project, I don't create Pull Requests within the fork.

The current issue I'm facing is that I want to test a branch in someone else's fork without actually cloning their fork.

As the person who owns the branch has not created a pull request in the upstream repository, I'm unable to easily switch to their branch from my fork which I have cloned.

As a solution, I have found it easy to create a Pull Request from their branch into *my* fork.

If I merge it into a branch in my fork, I can easily switch to it, but if I want it to be synced with the other person's branch (like a Pull Request would be) that won't work.

If the Pull Requests tab had listed pull requests appearing in my fork, I would be able to switch to that Pull Request easily and not just the upstream Pull Requests.
